src/components/Footer.astro

```diff
@@ -8,7 +8,11 @@ const year = new Date().getFullYear();
     <p class="text-center text-xs/relaxed text-gray-500 mt-8 mb-8">
       &copy; {year}. {config.site.title} - EV0. All rights REVERSED
       <span class="block mt-2">
-        Made with ♥ in Medellín, Colombia. by
+        Made with
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" class="w-3 h-3 inline-block">
```

Comment:
Please add the svg files as components in `components/icons`.

Reply:
sure :rocket: